### ISSUE TITLE
Script view does not display all fields on toggle

### ISSUE THREAD

**Issue by tmkarlsen** [cerpus/Edlib#1509]
**What's the issue**
Script view only seem to display general text fields. The following fields/filtering is missing:

- Alt text fields: missing/does not show alt text from images
- Hover fields - missing as it seems to display through "title" 
- Title - displays content from "Hover fields" in one case. Does not display titles.
- Header seems to be displayed as a task description field along with the other general content fields

**Steps to recreate**
1. Create a course presentation and add an image with alt text and hover text. Save and exit and edit it to see script view. Filter by all types. Only content fields and title fields will display something. Title filed will display hover text 
2. Create a documentation tool with several content elements. Only the content fields  will be displaying anything. The general header field for the task will be displayed as a "taskDescription" amongst the content fields, and not a header field.

**Comment by chrieinv** [comment]
The hover text for H5P Image displays as `title` because it's named `title` in the `semantics.json` file (https://github.com/h5p/h5p-image/blob/f753835c9e5d01b68962b435d7934ee14e5a7a61/semantics.json#L34).

The alt-text is filtered because it uses a widget, "showWhen". Allowing text fields that uses the `showWhen` widget would cause fields that should not be listed, for example some color picker values, to be listed.

In the Documentation Tool H5P the `Heading` fields is named `taskDescription` in the `semantics.json` file (https://github.com/h5p/h5p-documentation-tool/blob/c4419cf207a0bf584fa627965f65d800b7d355be/semantics.json#L3).

The filter in `Script view` tries to group the texts based on the info in the `semantics.json` file, The `Content fields` group contains texts that does not fall into any of the other groups.
